app/components/mobile-dialog.js

```diff
@@ -0,0 +1,66 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+export default class MobileDialogComponent extends Component {
+  @tracked toShow = this.checkDeviceType();
+
+  @action
+  closeDialog() {
+    this.toShow = false;
+  }
+
+  @action
+  openRDSApp() {
+    this.openApp();
+  }
+
+  checkDeviceType() {
+    let regexp = /android|iphone|kindle|ipad/i;
+    let details = navigator.userAgent;
```

Comment:
Can you please help me to understand about navigator.userAgent??

Reply:
UserAgent contains information about the browser name, version and platform you are using.